resources/knative-provisioner-natss/values.yaml

```diff
@@ -10,8 +10,8 @@ knative-provisioner-natss:
   controller:
     image:
       pullPolicy: IfNotPresent
-      location: gcr.io/knative-releases/github.com/knative/eventing/contrib/natss/cmd/channel_controller@sha256:41c06e6e20a66419bc63e75716390e0ef4d31f5d13f6d46efc34f31b0b420fd7
+      location: gcr.io/knative-releases/knative.dev/eventing-contrib/natss/cmd/channel_controller@sha256:3758dc790e862bac8098b469f3d37b4747fd6a5fb4e3a1f31bad1864e638c553
   dispatcher:
     image:
       pullPolicy: IfNotPresent
-      location: gcr.io/knative-releases/github.com/knative/eventing/contrib/natss/cmd/channel_dispatcher@sha256:757b9ab1042adefa3babbc85f7bbd9e7e38fe421198c42b442fe58df76ea14cd
+      location: gcr.io/knative-releases/knative.dev/eventing-contrib/natss/cmd/channel_dispatcher@sha256:cef2fbeff9e55b03b2b79e6b3e2db107645293cc2a86e3f25cf43b5df4b06f60
```

Comment:
@montaro please use the image from the knative eventing-contrib release 0.10.3.

Reply:
Done